src/plugins/discover/public/application/main/components/top_nav/open_search_panel.tsx

```diff
@@ -94,7 +94,9 @@ export function OpenSearchPanel(props: OpenSearchPanelProps) {
                 onClick={props.onClose}
                 data-test-subj="manageSearchesBtn"
                 href={addBasePath(
-                  `/app/management/kibana/objects?_a=${rison.encode({ tab: SEARCH_OBJECT_TYPE })}`
+                  `/app/management/kibana/objects?initialQuery=type:(${rison.encode(
+                    SEARCH_OBJECT_TYPE
+                  )})`
```

Comment:
```suggestion
                  `/app/management/kibana/objects?initialQuery=type:(${SEARCH_OBJECT_TYPE})`
```
I think we can skip using `rison.encode` here since `SEARCH_OBJECT_TYPE` is a simple string.